lib/rubocop/cop/rspec/pending_without_reason.rb

```diff
@@ -103,7 +103,7 @@ def on_send(node)
           on_pending_by_metadata(node)
           return unless (parent = parent_node(node))
 
-          if example_group?(parent) || block_node_example_group?(node)
+          if shared_parent?(parent) || block_node_example_group?(node)
```

Comment:
You can use [spec_group](https://github.com/rubocop/rubocop-rspec/blob/fb85f8819a94e6d7e70313d89bf2500366642415/lib/rubocop/rspec/language.rb#L38) here

Reply:
Updated!